packages/core/core/src/RequestTracker.js

```diff
@@ -979,6 +967,21 @@ async function loadRequestGraph(options): Async<RequestGraph> {
       snapshotPath,
       opts,
     );
+
+    for (let event of events) {
+      // If we see a 'create' event for the project root itself,
+      // this means the project root was moved and we need to
+      // re-run all requests.
+      if (event.path === options.projectRoot && event.type === 'create') {
+        for (let [id, node] of requestGraph.nodes) {
+          if (node.type === 'request') {
+            requestGraph.invalidNodeIds.add(id);
+          }
+        }
+        break;
```

Comment:
we can also return the requestGraph immediately here since everything's invalidated